plasmapy/formulary/radiation.py

```diff
@@ -31,7 +31,7 @@ def thermal_bremsstrahlung(
     n_e: u.m**-3,
     T_e: u.K,
     n_i: u.m**-3 = None,
-    ion_species: ParticleLike = "H+",
+    ion: ParticleLike = "p+",
```

Comment:
One thing to note here... `"H+"` ≠ `"p+"`.  `"H+"` gets its mass from the standard atomic weight of hydrogen in terrestrial environments and includes a fraction of deuterons, so it doesn't strictly correspond to a proton.